libwget/utils.c

```diff
@@ -62,6 +63,37 @@
  * They may be useful to other developers that is why they are exported.
  */
 
+static const char * _wget_ssl_default_path(BOOL bundleNotDir)
+{
+#ifndef _WIN32
+	return bundleNotDir ? NULL : "/etc/ssl/certs";
+#else
+	static char CERTDIR_PATH[MAX_PATH] = {0};

```

Comment:
A library function should not use `static` variables - except they are only initialized once during initialization phase.
Can we consider `getenv("ProgramData")` to be a constant ? If not, we should better pass this as a function argument. If yes, the function can be called just once in `tls_init()`.

Reply:
I think assuming programdata is a a constant is just fine, especially as I don't think anyone expects to be able to externally edit the ENV vars of a running program.   I favored initializing on demand as even if called simultaneously there was no race condition but will move the call to tls_init instead.